### DISCUSSION TITLE
Chracters won't equip Call to arms

### DISCUSSION THREAD

**Discussion by mirakhany** [isid0re/SoloLeveling#320] [Q&A] [answered]
Hi. my sorceress and druid made Call to arms with enough stats to equip.
but they won't equip Call to arms. only console message popup "equiping Crystal Sword" when they visit stash everytime.
please check this isusse.

**Comment by isid0re** [comment]
Thanks I will have this sorted soon

**Comment by isid0re** [comment, marked as answer]
this has been fixed.
